program/tests/helpers/mint_builder.rs

```diff
@@ -0,0 +1,226 @@
+use {
+    crate::helpers::common::{init_mollusk, setup_mint},
+    mollusk_svm::{result::Check, Mollusk},
+    solana_account::Account,
+    solana_program::system_program,
+    solana_pubkey::Pubkey,
+    spl_token_wrap::{
+        get_wrapped_mint_address, get_wrapped_mint_backpointer_address, instruction::create_mint,
+    },
+};
+
+pub struct CreateMintResult {
+    pub unwrapped_mint: KeyedAccount,
+    pub wrapped_mint: KeyedAccount,
+    pub wrapped_backpointer: KeyedAccount,
+}
+
+#[derive(Debug, Clone)]
+#[cfg(test)]
```

Comment:
nit: why does this have `cfg(test)`? It's all testing code, right? We might just need to add an `#[allow(dead_code)]` somewhere, potentially on the whole file

Reply:
Ah, a vestige of having `dead_code` linting issues and debugging. Will remove. Turns out I just needed to do this in `/tests/helpers/mod.rs`

```diff
- mod common;
+ pub mod common;
```